packages/webapp/public/locales/en/translation.json

```diff
@@ -500,10 +473,6 @@
     "ONE_UPPER_CASE": "at least one upper case character",
     "TOO_SHORT": "at least 8 characters"
   },
-  "ENTITY_TYPES": {
-    "LOCATION": "Location",
-    "TASK": "Task"
-  },
```

Comment:
This one we need, it's part of a dynamically defined string  (I guess the script will always fail with those -- 😔 sad because I just wrote one)

```
t(`ENTITY_TYPES.${entityType.toUpperCase()}`)
```

<img width="440" alt="Screenshot 2023-11-10 at 6 01 19 PM" src="https://github.com/LiteFarmOrg/LiteFarm/assets/82857964/c8321374-63fc-4109-9d93-e9768b55a27d">